Sming/Libraries/DS18S20/ds18s20.h

```diff
@@ -41,7 +41,7 @@ class DS18S20
 public:
     /** @brief  Instantiate a DS18S20 object
     */
-    DS18S20();
+    DS18S20(uint8_t);
```

Comment:
The default value for the constructor should be here and not in the cpp file. You should move `#define DS1820_WORK_PIN 2 ` here, change the signature of the constructor to be 
```
DS18S20(uint8_t workPin = DS1820_WORK_PIN);
```

And in the CPP file to have the following for the constructor:

```
DS18S20::DS18S20(uint8_t workPin /* = DS1820_WORK_PIN */)
{
	ds = ...
```